buildSrc/subprojects/documentation/src/main/groovy/gradlebuild/docs/GradleUserManualPlugin.java

```diff
@@ -160,7 +160,6 @@ private void generateUserManual(Project project, TaskContainer tasks, ProjectLay
 
             attributes.put("antManual", "https://ant.apache.org/manual");
             attributes.put("docsUrl", "https://docs.gradle.org");
-            attributes.put("guidesUrl", "https://guides.gradle.org");
```

Comment:
I intentionally removed this to make it clear that we don't wanna add more links to this.

There are still some links left in the user manual now that point to guides we want to move over to the user manual as separate pages.